cli/js/io.ts

```diff
@@ -104,3 +107,27 @@ export function toAsyncIterator(r: Reader): AsyncIterableIterator<Uint8Array> {
     },
   };
 }
+
+export function iterSync(
+  r: SyncReader,
+  bufSize?: number
+): IterableIterator<Uint8Array> {
```

Comment:
Should use generator syntax.

Reply:
What does that mean? Can you give an example?

Reply:
```ts
export function *iterSync(
  r: SyncReader,
  bufSize?: number
): IterableIterator<Uint8Array> {
  const b = new Uint8Array(bufSize ?? 1024);
  while (true) {
    const result = r.readSync(b);
    if (result === EOF) {
      break;
    }
    yield b.subarray(0, result);
  }
}
```